apps/financial-aid/web-osk/src/routes/application/Info/info.tsx

```diff
@@ -117,6 +145,20 @@ const ApplicationInfo = () => {
         </Text>
         <Text marginBottom={3}>Lögheimili, hjúskaparstaða</Text>
 
+        <Text as="h3" variant="h5" color="blue400">
+          Skatturinn
+        </Text>
+        <Text marginBottom={3}>
+          Afrit af skattframtali og upplýsingar um staðgreiðslu í
+          staðgreiðsluskrá.
+        </Text>
+
+        <Text marginBottom={3}>
+          Við þurfum að afla þessara gagna til að staðfesta hjá hvaða
+          sveitarfélagi þú eigir að sækja um fjárhagsaðstoð, einfalda umsóknar-
+          og vinnsluferlið og staðfesta réttleika upplýsinga.
+        </Text>
+
```

Comment:
Should all these texts be hardcoded/translatable ?

Reply:
Yes so the client is not localized and won't be. It was suppose to be future work (I would have love to have it straight away). But now there is a project in progress about moving this application into the application system where we will set up the localization. 😄